Dockerfile

```diff
@@ -23,7 +23,7 @@ COPY go.mod go.mod
 COPY go.sum go.sum
 # cache deps before building and copying source so that we don't need to re-download as much
 # and so that source changes don't invalidate our downloaded layer
-ENV GOPROXY https://goproxy.io,direct
+ENV GOPROXY https://proxy.golang.org,direct
```

Comment:
any particular reason for switching the proxy here? don't mind switching to this, but curious to know why :)

Reply:
> `GOPROXY` was changed due to the following error
> 
> ```
> Step 5/18 : ENV GOPROXY https://goproxy.io,direct                                                                                            
>  ---> Using cache                                                                                                                            
>  ---> feb3b3f1833e                                                                                                                           
> Step 6/18 : RUN go mod download                                                                                                              
>  ---> Running in a1bd6cc081e1                                                                                                                
> go mod download: github.com/smartystreets/assertions@v0.0.0-20180927180507-b2de0cb4f26d: verifying module: github.com/smartystreets/assertion
> s@v0.0.0-20180927180507-b2de0cb4f26d: Get "https://goproxy.io/sumdb/sum.golang.org/lookup/github.com/smartystreets/assertions@v0.0.0-20180927
> 180507-b2de0cb4f26d": read tcp 172.17.0.2:48652->119.28.201.50:443: read: connection reset by peer                                           
> go mod download: github.com/smartystreets/goconvey@v1.6.4: read tcp 172.17.0.2:48652->119.28.201.50:443: read: connection reset by peer      
> The command '/bin/sh -c go mod download' returned a non-zero code: 1                                                                         
> make: *** [Makefile:206: e2e-image] Error 1 
> ```

yes, as mentioned above, was seeing an issue on downloading a few go modules inspite of multiple retries. Switching the `GOPROXY` resolved it.